### PR TITLE
Update design.less

### PR DESCRIPTION
**Pull request by Dr-Yukon**
Added placeholder styles for search textarea

### CODE REVIEW

File: lib/tpl/dokuwiki/css/design.less [hunk 1]
```diff
@@ -206,6 +206,19 @@ form.search {
...
+}
+:focus::-moz-placeholder {
+color: transparent;
+}
+:focus:-moz-placeholder {
+color: transparent;
+}
+:focus:-ms-input-placeholder {
+color: transparent;
+}
```
Comment (splitbrain) [inline lib/tpl/dokuwiki/css/design.less]: indention is wrong here.

**Comment by splitbrain** [comment]
Can you explain why these are needed? Shouldn't the default browser styles take care of removing placeholders on focus?

**Comment by Dr-Yukon** [comment]
http://html5.by/blog/placeholder/

Not every browser can hide placeholder when user only click in textarea. Some browsers hide it when he begin to print.

**Comment by splitbrain** [comment]
I see. However I'm not convinced that we should change that default behaviour. After all it's what you would expect from your browser.

**Comment by mprins** [comment]
this PR actually makes the accessibilty of the whole placeholder thing worse, as it removes the placeholder on focus which is generally before people have had a chance to read it properly.

for reference:
  - [Placeholders in Form Fields Are Harmful](http://www.nngroup.com/articles/form-design-placeholders/)
  - [HTML5 Accessibility Chops: the placeholder attribute](http://www.paciellogroup.com/blog/2011/02/html5-accessibility-chops-the-placeholder-attribute/)

**Pull request closed without merging**